views/shared/layout.jsx

```diff
@@ -11,7 +11,7 @@ module.exports = function Layout(props) {
   return (
     <html lang="en">
       <head>
-        <meta charset="UTF-8" />
+        <meta charSet="UTF-8" />
```

Comment:
What is the difference between `charset` and `charSet`? I think HTML is case-insensitive

Reply:
React warnings told me to